lib/fathead/electron_api/output.txt

```diff
@@ -0,0 +1,590 @@
+app.setLoginItemSettings macOS Windows	A										<section class="prog__container"><ul>\n<li><code class="highlighter-rouge">settings</code> Object\n    <ul>\n<li><code class="highlighter-rouge">openAtLogin</code> Boolean (optional) - <code class="highlighter-rouge">true</code> to open the app at login, <code class="highlighter-rouge">false</code> to remove the app as a login item. Defaults to <code class="highlighter-rouge">false</code>.</li>\n<li><code class="highlighter-rouge">openAsHidden</code> Boolean (optional) - <code class="highlighter-rouge">true</code> to open the app as hidden. Defaults to <code class="highlighter-rouge">false</code>. The user can edit this setting from the System Preferences so <code class="highlighter-rouge">app.getLoginItemStatus().wasOpenedAsHidden</code> should be checked when the app is opened to know the current value. This setting is only supported on macOS.</li>\n</ul>\n</li>\n</ul><ul>\n<li><code class="highlighter-rouge">openAtLogin</code> Boolean (optional) - <code class="highlighter-rouge">true</code> to open the app at login, <code class="highlighter-rouge">false</code> to remove the app as a login item. Defaults to <code
```

Comment:
It looks like we should be stripping out the `Windows` and `macOS` from the titles -- it's not part of the function name.

We should however indicate which platform this applies to somewhere in the abstract. It could be the first line for example:

```html
<span class="prog__sub>Platform:</span>
<p>Windows</p>
```